frontend/src/features/idir/search/components/IDIRCompanySearchResults.tsx

```diff
@@ -76,7 +99,29 @@ export const IDIRCompanySearchResults = memo(
 
     // Column definitions for the table
     const columns = useMemo<MRT_ColumnDef<CompanyProfile>[]>(
-      () => CompanySearchResultColumnDef,
+      () => [
+        {
+          accessorKey: "legalName",
+          header: "Company Name",
+          enableSorting: true,
+          sortingFn: "alphanumeric",
+          Cell: (props: { row: any; cell: any }) => {
+            return (
+              <Link
```

Comment:
should this be `<CustomLink />`?

Reply:
CustomLink does not support an onClick. It is limited to just navigation.

Reply:
Turns out, CustomLink does support and it was my vscode that had to be restarted.